src/chunk.c

```diff
@@ -1934,7 +2237,7 @@ chunk_scan_find(int indexid, ScanKeyData scankey[], int nkeys, MemoryContext mct
 	return chunk;
 }
 
-Chunk *
+TSDLLEXPORT Chunk *
```

Comment:
Unnecessary change. Should be:
```suggestion
Chunk *
```